openmrs/src/main/java/org/motechproject/openmrs/domain/ProgramEnrollmentListResult.java

```diff
@@ -10,14 +10,24 @@
 
     private List<ProgramEnrollment> results;
 
+    private Integer numberOfPrograms;
+
     public List<ProgramEnrollment> getResults() {
         return results;
     }
 
+    public Integer getNumberOfPrograms() {
+        return numberOfPrograms;
```

Comment:
this field could be retrieved from results.size() rather than stored in private field.